easybuild/easyconfigs/f/FFTW3-sve/FFTW3-sve-1.0.0-ffmpi-21.05.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'EB_FFTW'
+
+name = 'FFTW3-sve'
```

Comment:
Wouldn't it make more sense to use `-fujitsu` versionsuffix here, since this is the Fujitsu fork of FFTW, rather than the upstream?

Reply:
yes, but won't `FFTW-1.0.0-ffmpi-21.05-fujitsu.eb` be confusing?

Reply:
No, I think that's the right way of naming it. The `version` and `versionsuffix` are a bit disconnected, sure, but I doubt we want to go and tweak the naming scheme for easyconfigs just because of this example. ;)